.env.defaults

```diff
@@ -34,11 +34,11 @@ CURVEBALL_ORIGIN="http://localhost:8531"
 
 # Postgres database settings
 #
-# DB_DRIVER=postgres
-# DB_HOST=127.0.0.1
-# DB_PASSWORD='password
-# DB_USER=a12nserver
-# DB_DATABASE=a12nserver
+DB_DRIVER=pg
+DB_HOST=127.0.0.1
+DB_PASSWORD=password
+DB_USER=a12nserver
+DB_DATABASE=a12nserver
```

Comment:
This should get reverted. The default right now is SQLite and commiting this would also break the `npx a12nserver` script

Reply:
oh yeah I didnt mean to commit this, I left them uncommented so you can try them!